packages/client-library-otel/package.json

```diff
@@ -47,9 +47,9 @@
     "build:types": "tsc --project tsconfig.json",
     "build:alias": "tsc-alias -p tsconfig.json -f",
     "build:swc": "cd src && swc . -d ../dist --source-maps",
-    "dev": "wrangler dev sample/index.ts",
+    "start-sample": "wrangler dev sample/index.ts",
     "prepublishOnly": "pnpm run build",
     "clean": "rimraf dist",
-    "watch": "nodemon --watch src --ext ts,js,json --exec \"pnpm run build\""
+    "dev": "nodemon --watch src --ext ts,js,json --exec \"pnpm run build\""
```

Comment:
How about something like: `build:watch` 

Also: isn't there a development.md file as well?

Reply:
oh this snuck in from an unrelated thing I was trying. Gonna just revert it